File: HELLO_PYTHON/day06/mysql_delete.py
import pymysql
 
conn = pymysql.connect(host='localhost', user='root', password='python',
                       port=3305, db='python', charset='utf8')
 
e_id = "5"

 
curs = conn.cursor()
sql = f"""
        delete from emp
        where e_id = '{e_id}'
"""
         
cnt = curs.execute(sql)
print(cnt)
# print(curs.rowcount)

conn.commit()

curs.close()
conn.close()

File: HELLO_NODE/mysql_delete.js
const mysql = require('mysql');
const connection = mysql.createConnection({
	host: 'localhost',
	port: 3305,
	user: 'root',
	password: 'python',
	database: 'python'
});

var e_id = '3';

connection.connect();
var sql = `
	delete from emp
		where e_id='${e_id}'
`;

connection.query(sql, (error, result) => {
	console.log(result.affectedRows);
});

connection.end();
